src/hooks/useSaveShortcut.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+
+const useSaveShortcut = (saveCallback: () => Promise<void>) => {
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
We should've probably documented it somewhere, but we have `keyboardjs` in our dependencies and it makes it much easier to manage shortcuts (here's an example: https://github.com/grafana/k6-studio/pull/177)